collect_app/src/main/java/org/odk/collect/android/activities/CaptureSelfieActivity.java

```diff
@@ -98,9 +118,22 @@ private Camera getCameraInstance() {
 
             if (camInfo.facing == Camera.CameraInfo.CAMERA_FACING_FRONT) {
                 camera = Camera.open(camNo);
+                cameraInfo = camInfo;
                 camera.setDisplayOrientation(90);
+
             }
         }
+
+        // Set the rotation of the camera which the output picture need.
+        if (camera != null) {
+            Camera.Parameters parameters = camera.getParameters();
+            int rotation = ORIENTATIONS.get(getWindowManager().getDefaultDisplay().getRotation());
+            parameters.setRotation(calcCameraRotation(rotation));
+            camera.setParameters(parameters);
+        } else {
+            Timber.e("No Available front camera");
```

Comment:
I'm wondering if it should be `Timber.e` it's not our fault if a device doesn't have a front camera but then it's visible in our reports as a fatal issue.